### DISCUSSION TITLE
[11 gennaio 2022 - A] Esercizio 5

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#197] [Esami]
**Esercizio 5 (4 punti)**

Analizzare la macchina a stati mostrata in figura. Scrivere le tabelle degli stati futuri e di uscita e disegnare l’automa (il diagramma di transizione degli stati).

![image](https://user-images.githubusercontent.com/69389461/211830694-d6506e14-8da6-4a66-b49f-6b82a4da1747.png)

**Comment by Elia-Belli** [comment]
- Lo stato $S0$ è irragiungibile, l'automa entrerà in quello stato solamente se i D fliflop verrano resettati a 00.
![AFSM](https://user-images.githubusercontent.com/107944476/212033331-30bb325c-9dbc-4b1c-bfe6-65eb5a964aab.jpg)

**Comment by fraxb04** [comment]
@Elia-Belli corretto
